src/components/Dropdown/Dropdown.tsx

```diff
@@ -71,31 +72,35 @@ const Dropdown: FC<DropdownProps> = ({
 
 	return (
 		<>
-			<div
-				className={rootCls}
-				onClick={() => toggle()}
-				onKeyPress={(e) => (e.key === 'Space' ? toggle() : () => null)}
-				role="button"
-				tabIndex={0}
-				ref={setReferenceElement}
-			>
-				{dropdownButtonSlot || (
-					<Button
-						iconStart={icon}
-						label={label}
-						iconEnd={isOpen ? iconOpen : iconClosed}
-					/>
-				)}
-			</div>
-
+			{
+				// Wrapper element should not be tabbable
+				// eslint-disable-next-line jsx-a11y/no-static-element-interactions
+				<div
+					className={rootCls}
+					onClick={() => toggle()}
+					onKeyPress={(e) => (e.key === 'Space' ? toggle() : () => null)}
+					ref={setReferenceElement}
+				>
```

Comment:
Zonder `tabIndex` kan dat element nooit gefocust worden en kan de `onKeyPress` toch niet runnen? Of komt dat via bubble-up door wanneer de onderliggende Button focus krijgt?

Reply:
Zolang je een button oid gebruikt in deze wrapper, dan zal dat bubbelen. Als dit tabbable is, dan kan je hier geen eigen focus styling aan meegeven of heb je twee opeenvolgende focus stops die hetzelfde doen (1x de wrapper en 1x je button)

Reply:
eigenlijk doet die `onKeyPress` hier helemaal niks 🤷